config/alpha.json

```diff
@@ -26,5 +26,6 @@
   "alphaSpendReportGroup": "Alpha_Spend_Report_Users",
   "alphaRegionalityGroup": "Alpha_Regionality_Users",
   "terraDockerVersionsFile": "terra-docker-versions-alpha.json",
-  "terraDockerImageBucket": "terra-docker-image-documentation-alpha"
+  "terraDockerImageBucket": "terra-docker-image-documentation-alpha",
+  "b2cBillingPolicy": "b2c_1a_signup_signin_billing_alpha"
```

Comment:
These policies exist in all non-prod envs as a result of https://github.com/broadinstitute/terraform-ap-deployments/pull/707